Dockerfile

```
@@ -33,7 +33,7 @@ WORKDIR /workdir
 
 # Tell git it's safe to access /workdir/.git even if
 # the directory is owned by a different user
-RUN git config --global --add safe.directory /workdir
+RUN git config --global --add safe.directory /workdir/.git
```

May be clearer to opt out entirely:
```suggestion
RUN git config --global --add safe.directory '*'
```

---

Can't spot any ownership mismatches:

```
laborant@dev-machine:~/svc-a$ ls -lah
total 28K
drwxr-xr-x 5 laborant laborant 4.0K Sep 28 11:39 .
drwxr-x--- 9 laborant laborant 4.0K Sep 28 15:54 ..
-rw-r--r-- 1 laborant laborant   29 Sep 28 10:35 .env
drwxr-xr-x 8 laborant laborant 4.0K Sep 28 11:39 .git
drwxr-xr-x 3 laborant laborant 4.0K Sep 28 10:41 .kamal
-rw-rw-r-- 1 laborant laborant  531 Aug 28 21:07 Dockerfile
drwxr-xr-x 2 laborant laborant 4.0K Sep 28 11:35 config
laborant@dev-machine:~/svc-a$ ls -lah .git/
total 52K
drwxr-xr-x  8 laborant laborant 4.0K Sep 28 11:39 .
drwxr-xr-x  5 laborant laborant 4.0K Sep 28 11:39 ..
-rw-r--r--  1 laborant laborant    8 Sep 28 11:39 COMMIT_EDITMSG
-rw-r--r--  1 laborant laborant   21 Sep 28 11:39 HEAD
drwxr-xr-x  2 laborant laborant 4.0K Sep 28 11:39 branches
-rw-r--r--  1 laborant laborant   92 Sep 28 11:39 config
-rw-r--r--  1 laborant laborant   73 Sep 28 11:39 description
drwxr-xr-x  2 laborant laborant 4.0K Sep 28 11:39 hooks
-rw-r--r--  1 laborant laborant  853 Sep 28 11:39 index
drwxr-xr-x  2 laborant laborant 4.0K Sep 28 11:39 info
drwxr-xr-x  3 laborant laborant 4.0K Sep 28 11:39 logs
drwxr-xr-x 17 laborant laborant 4.0K Sep 28 11:39 objects
drwxr-xr-x  4 laborant laborant 4.0K Sep 28 11:39 refs
```

You can try it out yourself - these Kamal VMs are publicly accessible.

I don't mind changing it to `'*'` - it's probably still safe enough given the context of usage.

---

Perhaps the relaxed version should be released as `v1.9.2` as well. While upgrading, I had to use the following to make the deployment work on Linux.

```
docker build -t ghcr.io/basecamp/kamal:v1.9.1-fix - <<EOF
FROM ghcr.io/basecamp/kamal:v1.9.1

RUN git config --global --add safe.directory '*'
EOF
```

`alias kamal='docker run -it --rm -v "${PWD}:/workdir" -v "${SSH_AUTH_SOCK}:/ssh-agent" -v /var/run/docker.sock:/var/run/docker.sock -e "SSH_AUTH_SOCK=/ssh-agent" ghcr.io/basecamp/kamal:v1.9.1-fix'`

This was the error on `v1.9.1`

```
ERROR (SSHKit::Command::Failed): git exit status: 32768
git stdout: Nothing written
git stderr: Cloning into 'workdir'...
fatal: detected dubious ownership in repository at '/workdir/.git'
To add an exception for this directory, call:

	git config --global --add safe.directory /workdir/.git
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

---

It actually makes a lot of sense because it seems like the v1 line is currently broken, and not everyone is in a position to upgrade to v2 right away.